docs/monitoring.md

```diff
@@ -435,6 +435,7 @@ set of sinks to which metrics are reported. The following instances are currentl
 * `executor`: A Spark executor.
 * `driver`: The Spark driver process (the process in which your SparkContext is created).
 * `shuffleService`: The Spark shuffle service.
+* `yarn`: Spark resource allocations on YARN.
```

Comment:
Is it better to change to application master for better understanding?

Reply:
Sure we can do that. After this many change I would like to test it on a cluster again. 
Soon I will come back with the result.

Reply:
Successfully retested on cluster.